store/errors.go

```diff
@@ -88,3 +88,16 @@ func NewErrNotFound(resource, id string) *ErrNotFound {
 func (e *ErrNotFound) Error() string {
 	return "resource: " + e.resource + " id: " + e.Id
 }
+
+// ErrOutOfBounds indicates that the requested total numbers of rows was greater than allowed
```

Comment:
"... was greater than the allowed limit".

Reply:
yep, my english